Nadim.CinemaReservationSystem.Web/ClientApp/src/components/Authentication/Logout.js

```diff
@@ -1,26 +1,26 @@
 import React, { Component } from 'react';
 import { Button } from 'react-bootstrap';
-import authorizationService from '../../Services/AuthorizationService';
+import userService from '../../Services/UserService';
 
 export default class Logout extends Component{
     displayName = Logout.displayName;
 
     handleLogout = () =>{
-        authorizationService.setInfo('', '', '', '');
+        userService.setInfo('', '', '', '');
```

Comment:
You should create Logout method in UserService and call it here. Login component (and any client of UserService for that matter) shouldn't know what exact information is stored in UserService. Then you can (and should) make setInfo method private or even get rid of it completely.